app/src/utils/constants.ts

```diff
@@ -34,12 +36,18 @@ export const ERC20_ABI = [
 
 export const LOREM_IPSOM_TEXT = 'A brief description of your grant goes here.\n\nLorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.'; // prettier-ignore
 
-// LocalStorage keys
+// LocalForage config
+export const localForageConfig: LocalForageConfig = {
+  name: 'dGrants',
+  version: 1,
```

Comment:
Should this correspond to the `package.json` version number, in which case we can import that like we do in the `About` component, or is this separate?

Reply:
This version is used in `indexedDb` to version the db structure and allow for a migration phase, I think in `localForage` we can still use it like that and force cache-busting on the end-users if we bump the version.

I've added some comments inline to explain this thinking 👍  _(to be tested!)_